src/cryptography/x509/graph.py

```diff
@@ -0,0 +1,55 @@
+# This file is dual licensed under the terms of the Apache License, Version
+# 2.0, and the BSD License. See the LICENSE file in the root of this repository
+# for complete details.
+
+from __future__ import absolute_import, division, print_function
+
+
+class Graph:
```

Comment:
```suggestion
class Graph(object):
```

If python2 support is not dropped yet, I believe we should use the new-style class format.